### DISCUSSION TITLE
exec --no-startup-id doesnt work

### DISCUSSION THREAD

**Discussion by Filip-Nachov** [i3/i3#6124] [Q&A] [answered]
I am simply just trying to config i3 so it starts wal -R on startup with exec --no-startup-id as that is the way I did the start up of picom and redshift, but it doesnt work for wal for some reason please help

**Comment by okraits** [comment, marked as answer]
Does it work if you run the same command in a terminal?

**Reply by Filip-Nachov** [reply]
it does the it is not the problem the problem is that it cant run terminal commands so I had to run it in the .bashrc
